src/engine.h

```diff
@@ -77,6 +77,12 @@ class EngineController {
   void PonderHit();
   // Must not block.
   void Stop();
+  // Prints verbose move stats of current root
+  void Stats() {
+    // Reserved for debugging
+    //SetupPosition(current_position_->fen, current_position_->moves);
```

Comment:
Why is that commented out? And what it is supposed to do? Maybe make `go nodes 0` to do that instead?

Reply:
Deleted.